src/styles/blocks/header.scss

```diff
@@ -0,0 +1,70 @@
+@import "../utils/variables";
```

Comment:
no need to import it into each file
import it (and other utils) in `main.scss` **before** block imports